pom.xml

```
@@ -482,7 +482,7 @@
             <configuration>
               <rules>
                 <requireJavaVersion>
-                  <version>[11.0,12.0)</version>
+                  <version>[11.0,12.0),[17.0,18.0)</version>
```

It's there because we want to make sure it is released using Java 11, see https://github.com/openhab/openhab-distro/issues/1540.

```suggestion
                  <version>[11.0,12.0)</version>
```